EasyPost.NetFramework/Security.cs

```diff
@@ -11,7 +11,7 @@ internal class Security
         public static SecurityProtocolType GetProtocol()
         {
 #if NET45
-            return SecurityProtocolType.Tls | SecurityProtocolType.Tls11 | SecurityProtocolType.Tls12;
+            return SecurityProtocolType.Tls;
```

Comment:
Is everything wrapped up under the `Tls` umbrella now? What happens by removing 1.1 and 1.2?

Reply:
The `Tls11` and `Tls12` enums don't exist in .NET Framework 3.5, everything falls under `Tls`

Reply:
This should not affect functionality, since it was an `or` clause to begin with.